app/delivery/send_to_providers.py

```diff
@@ -336,16 +339,55 @@ def update_notification_to_sending(notification, provider):
     dao_update_notification(notification)
 
 
-def provider_to_use(notification_type, notification_id, international=False, sender=None, template_id=None):
-    # Temporary redirect setup for template IDs that are meant for the short code usage.
-    if notification_type == SMS_TYPE and template_id is not None and str(template_id) in Config.AWS_PINPOINT_SC_TEMPLATE_IDS:
-        return clients.get_client_by_name_and_type("pinpoint", SMS_TYPE)
+def provider_to_use(
+    notification_type: str,
+    notification_id: UUID,
+    to: Union[str, None] = None,
```

Comment:
A more up to date syntax or type might be (after a proper `import typing.Optional`):
```suggestion
    to: Optional[str] = None,
```
Unsure for Python 3.10 but I think you can even go with:

```
    to: str | None = None,
```

But I personally prefer the `Optional` wrapper type in this case instead of the syntactic sugar version, might be preference, but definitely favored over `Union` with a `None` type though.